core/src/commonTest/kotlin/SharedRepeatableActionTests.kt

```diff
@@ -127,4 +129,21 @@ class SharedRepeatableActionTests {
 
         coroutineContext.cancelChildren()
     }
+
+    @Test
+    fun nothingLaunchedFromScope_remainsActive() = runTest {
```

Comment:
Total nit, but it'd be nice to have `sharedRepeatableAction` specified in the signature as the thing being tested. Understandably, we get this information through the class name, so I can understand why it's left out. But being verbose is free. ^_^